code/parachain/frame/composable-traits/src/fnft.rs

```diff
@@ -56,6 +56,7 @@ pub trait FinancialNftProtocol {
 	/// - collection: id of the financial NFT collection issued/used by the protocol.
 	/// TODO (vim): Think how to represent the difference between assets and liabilities
 	fn value_of(
+		pool_id: &Self::AssetId,
```

Comment:
This must not be required, breaks the interface if third parties who has no idea about the pools to find a pool id

Reply:
I see; I'll make a workaround so the interface can still be neutral to the underlying design